corehq/apps/cleanup/management/commands/couch_integrity/basic.json

```diff
@@ -0,0 +1,19 @@
+{
+    "databases": [
+        {
+            "uri": "",
```

Comment:
can you add an example of what should go here (assume is blank because we didn't want to expose credentials)?

it would be nice if we could just pull the credentials from `settings.py` somehow so that we can keep a basic check in the repo that can be run. but happy to get push back on that if you disagree.